cores/esp8266/core_esp8266_phy.c

```diff
@@ -88,7 +88,11 @@ static const uint8_t ICACHE_FLASH_ATTR phy_init_data[128] =
     // 0: 40MHz
     // 1: 26MHz
     // 2: 24MHz
+    #ifdef F_CRYSTAL
```

Comment:
Fair enough, let's keep this menu for `generic`. Other boards can use 26MHz via silent default.